zephyr-make

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/make -rf
+
+BOARD ?= qemu_x86
+CONF_FILE = prj.conf
+
+TARGETS = run flash debug debugserver ram_report rom_report
```

Comment:
This will confuse a user when he tries to use a target that is supported by CMake, but is not in this list.

Reply:
Such needs to be added (manually) and shared "upstream" (i.e. in this patch). Given that the aim of this script is to build Zephyr "like before", as long as "before" targets are there, it should be good enough (I didn't need any other target so far I believe).